examples/get-stream-settings/src/main.rs

```diff
@@ -0,0 +1,34 @@
+// example for reading stream settings
+
+use arctic::H10MeasurementType;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let mut polar = arctic::PolarSensor::new("9F8F7521".to_string())
+        .await
+        .unwrap();
+
+    while !polar.is_connected().await {
+        match polar.connect().await {
+            Err(arctic::Error::NoBleAdaptor) => {
+                println!("No bluetooth adapter found");
+                return Ok(());
+            }
+            Err(why) => println!("Could not connect: {:?}", why),
```

Comment:
Can you add language either here or above that it's polling for a connection? Because I hadn't run this in a while I forgot that this sort of thing may be required and thought that it wasn't connecting at all for a few runs.

Reply:
For sure